app/models/normalize_timdex_record.rb

```diff
@@ -0,0 +1,140 @@
+# Transforms a TIMDEX result into a normalized record.
+class NormalizeTimdexRecord
+  def initialize(record, query)
+    @record = record
+    @query = query
+  end
+
+  def normalize
+    {
+      # Core fields
+      'title' => title,
+      'creators' => creators,
+      'source' => source,
+      'year' => year,
+      'format' => format,
+      'links' => links,
+      'citation' => citation,
+      'identifier' => record_id,
```

Comment:
I'd prefer if we had the same normalized key as our method name. I think this is the only one that isn't consistent.

I believe we could rewrite this hash to look something like:

```ruby
{
  title:,
  creators:,
  ... etc
}
```

Moving to symbols does change the views probably, but it's kinda neat how it can automatically call the appropriate method based on convention.

Reply:
That's definitely a sleeker syntax! But yeah, it will wreak havoc elsewhere. I've already run into pain with string versus symbol keys, as I always seem to do when working with this repo. I opened [this ticket](https://mitlibraries.atlassian.net/browse/USE-109) to implement the change.